webapi/src/test/scala/org/knora/webapi/e2e/v2/OntologyV2R2RSpec.scala

```diff
@@ -78,7 +78,20 @@ class OntologyV2R2RSpec extends R2RSpec {
   private val clientTestDataPath: Seq[String] = Seq("v2", "ontologies")
 
   // Collects client test data
-  private val clientTestDataCollector = new ClientTestDataCollector(settings)
+  // TODO: redefine below method somewhere else if can be reused over other test files
+  private def CollectClientTestData(filename: String, payload: String) = {
```

Comment:
```suggestion
  private def collectClientTestData(filename: String, filecontent: String) = {
```

Reply:
@irinaschubert  at the end it's both payload and the file content ;) but in the matter of consistency I accept this suggestion.

`val clientTestDataCollector = new ClientTestDataCollector(settings)` can be also defined outside the method, maybe this is better performance wise. Everything was working well, but I think that some tests failed after that commit 2d9c26d where we added 3 missing test data.